cardano-node/src/Cardano/Tracing/OrphanInstances/Common.hs

```diff
@@ -58,7 +58,7 @@ import           Cardano.BM.Stats
 import           Cardano.BM.Tracing (HasPrivacyAnnotation (..), HasSeverityAnnotation (..),
                    Severity (..), ToObject (..), Tracer (..), TracingVerbosity (..),
                    Transformable (..))
-import           Cardano.Slotting.Block (BlockNo (..))
+import           Cardano.Node.Handlers.Shutdown ()
```

Comment:
Why remove an orphan instance from here? We should leave orphan instances  here.

Reply:
@Jimbo4350, because it turned out there's _another_ pre-existing orphan instance elsewhere -- so I just reused that.

Reply:
My apologies. I thought you moved it to another module.